dev/io.openliberty.netty.internal.impl/src/io/openliberty/netty/internal/impl/NettyFrameworkImpl.java

```diff
@@ -368,4 +405,9 @@ private void logChannelStopped(Channel channel) {
             }
```

Comment:
what if we are epol or something else in the future - do we want a 'vanilla' catch all log msg?